js/store.js

```diff
@@ -144,16 +149,11 @@ const store = {
       hostname: hostname,
       favicon: website.faviconUrl || '',
       firstPartyHostnames: website.firstPartyHostnames || false,
-      firstParty: 0,
+      firstParty: !!website.firstParty,
```

Comment:
`website.firstParty` is coming in as 1 and 0 instead of true and false due to `setWebsite` now "munging" the data, so you use the `!!` to coerce the value into a boolean data type in `outputWebsite` for the visualization.